packages/openneuro-components/src/search-page/SearchPageContainerExample.tsx

```diff
@@ -0,0 +1,203 @@
+import { sortBy } from '../mock-content/sortby-list'
+import { FacetSelect } from '../facets/FacetSelect'
+import { FacetRadio } from '../facets/FacetRadio'
+import { FacetRange } from '../facets/FacetRange'
+
+import {
+  modalities,
+  show_available,
+  showMyUploads_available,
+  diagnosis,
+  task,
+  author_pi,
+  gender,
+  species,
+  section,
+  domain,
+} from '../mock-content/facet-content'
+import { AccordionWrap } from '../accordion/AccordionWrap'
+import { AccordionTab } from '../accordion/AccordionTab'
+
+import './search-page.scss'
+
+export interface SearchContainereProps {
+  portalContent?: Record<string, any>
+  searchResults
+  profile?: Record<string, any>
+}
+
+export const SearchPageContainerExample = ({
+  searchResults,
+  portalContent,
+  profile,
+}: SearchContainereProps) => {
+  const [selected, setSelected] = React.useState()
+  const [active, setActive] = React.useState(0)
+  const [newvalue, setNewValue] = React.useState([0, 20])
+  console.log(active)
+  return (
+    <div>
+      <SearchPage
+        portalContent={portalContent}
+        renderSortBy={() => (
+          <>
+            <div className="col">
+              <b>
+                100 Datasets found for "<span>MRI</span>"
+              </b>
+            </div>
+            <div className="col">
+              <div className="search-sort">
+                <SearchSortContainerExample items={sortBy} />
+              </div>
+            </div>
+          </>
+        )}
+        renderSearchFacets={() => (
+          <>
+            <KeywordInputContainerExample searchValue={'MRI'} />
+            <FiltersBlock />
+            <FacetBlockContainerExample>
+              <FacetSelect
+                selected={selected}
+                setSelected={setSelected}
+                items={modalities}
+                accordionStyle="plain"
+                label="Modalities"
+                startOpen={false}
+              />
+
+              <FacetRadio
+                radioArr={show_available}
+                layout="row"
+                name="show-datasets"
+                startOpen={false}
+                label="Show"
+                accordionStyle="plain"
+                active={active}
+                setActive={setActive}
+              />
+
+              {active == 2 ? (
+                <FacetSelect
+                  selected={selected}
+                  setSelected={setSelected}
+                  items={showMyUploads_available}
+                  accordionStyle="plain"
+                  label="My Datasets Status"
+                  startOpen={false}
+                />
+              ) : null}
+
+              <FacetRange
+                startOpen={false}
+                label="Age"
+                accordionStyle="plain"
+                min={0}
+                max={100}
+                step={10}
+                dots={true}
+                pushable={5}
```

Comment:
This is throwing an error because pushable is defined as a boolean, but would need to be `boolean | number` in TwoHandleRange and Range to set the minimum ensured distance. The weird thing is that [rc-slider defines it as only a boolean](https://github.com/react-component/slider/blob/af77a4c7fb3ce22809b03e56027538cecff11601/src/Range.tsx#L42). There's a [PR](https://github.com/react-component/slider/pull/725) open for it, but until it gets merged, we'll have to use the weird workaround.
```suggestion
                pushable={(5 as unknown) as undefined}
```
and `props.pushable` in the Range components set as optional.